common/network-common/src/main/java/org/apache/spark/network/util/JavaUtils.java

```diff
@@ -226,6 +226,14 @@ private static boolean isSymlink(File file) throws IOException {
    * The unit is also considered the default if the given string does not specify a unit.
    */
   public static long timeStringAs(String str, TimeUnit unit) {
+    return timeStringAs(str, unit, unit);
+  }
+
+  /**
+   * Convert a passed time string (e.g. 50s, 100ms, or 250us) to a time count in the given unit.
+   * defaultUnit is used for string which have just number and no units mentioned
+   */
+  public static long timeStringAs(String str, TimeUnit unit, TimeUnit defaultUnit) {
```

Comment:
I suggest some renaming:
- unit => targetUnit
- defaultUnit => defaultSourceUnit

Reply:
This method is dropped after review